tests/conftest.py

```diff
@@ -27,3 +29,13 @@ def disable_autosave():
         settings.SettingsBase, "register_for_autosave"
     ) as register:
         yield register
+
+
```

Comment:
New fixture turns on the random app name environment variable for automated testing and creates it for the entire test run.